projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -38,35 +38,19 @@
 		function getAverageColor( imgEl ) {
 			var canvas = document.createElement( 'canvas' ),
 				context = canvas.getContext && canvas.getContext( '2d' ),
-				imgData,
-				width,
-				height,
-				length,
-				rgb = { r: 0, g: 0, b: 0 },
-				count = 0;
+				rgb = { r: 0, g: 0, b: 0 };
 
 			if ( ! imgEl ) {
 				return rgb;
 			}
-
-			height = canvas.height = imgEl.naturalHeight || imgEl.offsetHeight || imgEl.height;
-			width = canvas.width = imgEl.naturalWidth || imgEl.offsetWidth || imgEl.width;
-
+			imgEl.crossOrigin = 'Anonymous';
```

Comment:
I think we should pull #20485 into this PR since we're already setting the `crossOrigin` here so we can test. This should be updated with a try/catch per the comment on that PR https://github.com/Automattic/jetpack/pull/20485#discussion_r679593376